main.js

```diff
@@ -90,6 +89,48 @@ const renderMainWindow = () => {
         });
     }
   })();
+  
+  //read file and show it in app
+  ipcMain.handle('request-load-text', () => {
+    return fs.readFileSync(filePath, "utf8");
+  });
+
+  function getMasterPassword() {
+    return JSON.parse(fs.readFileSync(passwordFilePath, 'utf-8'));
+  }
+  
+  ipcMain.on('save-and-encrypt-text', async (event, text) => {
+    const userFilesDir = path.dirname(process.env.USER_FILE_PATH);
+    const filePath = path.resolve(process.env.USER_FILE_PATH);
+    const encryptedFilePath = path.join(userFilesDir, 'encrypted.txt');
+  
+    try {
+      fs.writeFileSync(filePath, text, 'utf8');
```

Comment:
it doesn't make sense, save not encrypted plain text in file, always save only encrypted